backends/advanced/src/advanced_omi_backend/services/transcription/__init__.py

```
@@ -42,67 +45,43 @@ def get_transcription_provider(
-    else:
-        return None
+    try:
+        cfg = _load_root_config() or {}
+        defaults = cfg.get("defaults", {}) or {}
+        models = cfg.get("models", []) or []
+        stt_name = defaults.get("stt")
+        if stt_name:
+            stt_def = next((m for m in models if m.get("name") == stt_name), None)
+            if stt_def:
+                provider = (stt_def.get("model_provider") or "").lower()
+                api_family = (stt_def.get("api_family") or "http").lower()
```

Ruff (0.14.8)

Local variable `api_family` is assigned to but never used

Remove assignment to unused variable `api_family`

(F841)